Dockerfile

```
@@ -0,0 +1,10 @@
+FROM node:latest
```

Hadolint

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)